### PR TITLE
Add data from Han et al. (2020).

### PR DESCRIPTION
**Pull request by tillahoffmann**
Addition of data from https://shedding.readthedocs.io/en/latest/publications/Han2020/Han2020.html#han-et-al-2020.

### CODE REVIEW

File: data/han2020sequential/han2020sequential.yaml [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+  Sequential Analysis of Viral Load in a Neonate and Her Mother Infected With
+  Severe Acute Respiratory Syndrome Coronavirus 2
+description:
+  The study reports SARS-CoV-2 viral loads in different specimen types for a neonate and
+  her mother diagnosed on 2020-03-20. The study includes nasopharyngeal, oropharyngeal,
+  stool, plasma, saliva, and urine samples. Only concentrations in stool samples are
+  reported here. Viral loads were extracted manually from Figure 1 using
+  [WebPlotDigitizer](https://automeris.io).
+doi: 10.1093/cid/ciaa447
+analytes:
```
Comment (YWAN446) [inline data/han2020sequential/han2020sequential.yaml:12]: This paper has nasopharynx, oropharynx, stool, saliva, plasma, and urine samples. Do we also want to include other sample types? One student can help with the extraction.

File: data/han2020sequential/han2020sequential.yaml [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+    limit_of_detection: 5700
+    limit_of_quantification: unknown
+    gene_target: E and ORF1b
+    biomarker: SARS-CoV-2
+    reference_event: symptom onset
+    specimen: stool
+    unit: gc/mL
+participants:
+  - attributes:
+      age: 0
```
Comment (YWAN446) [inline data/han2020sequential/han2020sequential.yaml:28]: 27 day old. Should we enter 0.074 (27 days/365 days) or 0.083 (1 month/12 months) here?

**Review by YWAN446 — COMMENTED** [review]
We can ask one student to help add results from the other sample types.

File: data/han2020sequential/han2020sequential.yaml [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+analytes:
+  stool_E:
+    description: Viral RNA was detected using the PowerChek 2019-nCoV real-time
+      polymerase chain reaction kit (Kogene Biotech, Seoul, Korea) for amplification of
+      the E gene and the RNA-dependent RNA polymerase region of the ORF1b gene, and
+      quantified with a standard curve that was constructed using in vitro transcribed
+      RNA provided from the European Virus Archive. [copied from manuscript]
+    limit_of_detection: 5700
+    limit_of_quantification: unknown
+    gene_target: E and ORF1b
```
[the remark this answers is not in the document]
Comment (tillahoffmann) [reply data/han2020sequential/han2020sequential.yaml:21]: Nice catch, I'll update that!

File: data/han2020sequential/han2020sequential.yaml (hunk 2 above)
Comment (tillahoffmann) [reply data/han2020sequential/han2020sequential.yaml:28]: Yes, let's do that. Can't hurt, I guess. I couldn't find an age for the mother.

File: data/han2020sequential/han2020sequential.yaml (hunk 1 above)
Comment (tillahoffmann) [reply data/han2020sequential/han2020sequential.yaml:12]: Yes, I think that would be nice to include. Maybe we can do that in a follow-up PR?

Comment (tillahoffmann) [reply data/han2020sequential/han2020sequential.yaml:12]: I've added an issue at #57.

**Comment by tillahoffmann** [comment]
Are we ready to merge this one, @YWAN446?

**Review by YWAN446 — APPROVED** [review]

**Pull request merged**